src/main/java/org/codeforamerica/shiba/output/caf/NonHourlyJobIncomeInformation.java

```diff
@@ -23,10 +24,10 @@ public boolean isComplete() {
     }
 
     @Override
-    public Double grossMonthlyIncome() {
+    public Money grossMonthlyIncome() {
         return switch (payPeriod) {
-            case EVERY_WEEK -> incomePerPayPeriod * 4;
-            case EVERY_TWO_WEEKS, TWICE_A_MONTH -> incomePerPayPeriod * 2;
+            case EVERY_WEEK -> incomePerPayPeriod.multiply(new Money(4));
+            case EVERY_TWO_WEEKS, TWICE_A_MONTH -> incomePerPayPeriod.multiply(new Money(2));
```

Comment:
`new Money(4)` and `new Money(2)` could be constants too

Reply:
`BigDecimal` has `ZERO`, `ONE` and `TEN` so I decided not to add these as constants to match that interface.